srml/kton/src/mock.rs

```diff
@@ -1,27 +1,33 @@
+pub use node_runtime::constants::currency::MILLICENTS;
```

Comment:
It is not good idea for srml to dependent on node_runtime.

Because if some other projects (or runtime) want to use this srml, they will have to dependent on darwinia runtime.

Reply:
No, this only define in `[dev-dependencies]`. Just for testing env.